resource_activity/models/resource_available.py

```diff
@@ -11,9 +12,37 @@ class ResourceAvailable(models.Model):
     _name = "resource.available"
     _description = "Resource Available"
 
+    @api.depends("allocation_id", "allocation_id.resource_id")
+    def _compute_resource_id(self):
+        for record in self:
```

Comment:
ah, t'es finalement convaincu par les `for record in self` toi ?

Reply:
I think they (Odoo) like to have the same loop everywhere, I guess it allows to have autogenerated loop code easier. Personally, I prefer to know what I'm looping on and have a clearer name.

Reply:
it’s a matter of taste, i guess. i prefer a more generic name, like with `self`. with a different name, i’m always asking myself whether we’re referring to something else.